terraform/demo/main.tf

```diff
@@ -48,3 +62,16 @@ module "lev_api" {
   }
   environment_name = "demo"
 }
+
+
+module "data-share-service" {
+  source = "../modules/data-share-service"
+  providers = {
+    aws.us-east-1 = aws.us-east-1
+  }
+  environment                 = "demo"
+  ecr_url                     = "${data.aws_caller_identity.current.account_id}.dkr.ecr.eu-west-2.amazonaws.com"
```

Comment:
I _think_ I'd like this to come out of the ecr module as an output, but lets not worry about it for now (it'll make the non-demo environment trickier anyway)

Reply:
Yeah tracking that across to non demo is an issue.

Also all it is is the account number followed by a constant string, so it is less important to output

Reply:
I think we should be able to do some remote state/similar, but lets not worry for now